packages/web-frontend/src/containers/OverlayDiv/components/LandingPage/LoginPage.js

```diff
@@ -44,6 +44,11 @@ const Tabs = styled(MuiTabs)`
   }
 `;
 
+const TabContentWrapper = styled(Box)`
+  @media screen and (min-width: ${({ theme }) => theme.breakpoints.values.sm}px) {
+    padding: 1.5em;
+  }
+`;
```

Comment:
Reduced padding here for small sized screens so the login modal fits in small screens